### PR TITLE
Pretty wkt output

### PR DESCRIPTION
**Pull request by OrangeOranges**
## Description

Adds syntax highlighting to well-known text output in function get_meta (when writing to a terminal).

Existing tests pass.
Checked syntax highlighting manually in terminal.

### CODE REVIEW

File: sno/output_util.py [hunk 1]
```diff
@@ -77,7 +79,47 @@ def format_json_for_output(output, fp, json_style="pretty"):
         return json.dumps(output, **JSON_PARAMS[json_style]) + "\n"
 
 
-def wrap_text_to_terminal(text, indent=''):
+def format_wkt_for_output(output, fp):
+    """
+    If the given filelike object is a terminal, adds whitespace and syntax highlighting to the output.
```
Comment (craigds) [inline sno/output_util.py:84]: IMO the whitespace is useful even if it's not going to a terminal

**Review by craigds — COMMENTED** [review]
screenshot from `sno meta get` output:

Screen Shot 2020-09-03 at 3 23 38 PM


looks nice!

I think it needs a quick test for the whitespace changes at least. i.e. just invoke `sno meta get <datasetname> metadata/dataset.json` and check the output has the expected whitespacing :)

**Comment by craigds** [comment]
macOS build failure appears to be a code-signing thing; I guess it doesn't work on third-party PRs. Can ignore that

**Comment by rcoup** [comment]
> macOS build failure appears to be a code-signing thing; I guess it doesn't work on third-party PRs. Can ignore that

Should skip rather than failing, I'll get it fixed on master
